Dockerfile

```diff
@@ -6,19 +6,33 @@ RUN pip install poetry
 WORKDIR /DevOps-Course-Starter
 COPY poetry.lock pyproject.toml /DevOps-Course-Starter/
 
-RUN poetry config virtualenvs.create false
+RUN poetry config virtualenvs.create false --local && poetry install
 
 COPY . /DevOps-Course-Starter/
 
-EXPOSE 5000
+#EXPOSE 5000
 
 FROM base as production
 ENV FLASK_ENV=production
 RUN poetry install --no-dev
-ENTRYPOINT ["gunicorn", "--config", "gunicorn.conf.py", "wsgi:app"]
```

Comment:
Your README is now out of date because it says you can run `docker run --env-file .env -p 5000:5000 todo-app:prod`

You can fix it by setting a default value for $PORT: `--bind 0.0.0.0:${PORT:-5000}`

Reply:
If you're no longer using gunicorn.conf.py, delete it.

If you wanted to keep using but found it didn't work with $PORT - it's a python file, so it needs to load environment variables the same way as any other python file: `os.getenv(...)` or `os.environ[...]`. You can't just do $PORT